.rubocop_todo.yml

```diff
@@ -305,24 +454,959 @@ Style/CommentedKeyword:
   Exclude:
     - 'app/controllers/application_controller.rb'
 
-# Offense count: 2
+# Offense count: 1
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle, SingleLineConditionsOnly, IncludeTernaryExpressions.
 # SupportedStyles: assign_to_condition, assign_inside_condition
 Style/ConditionalAssignment:
   Exclude:
     - 'app/controllers/api/taxons_controller.rb'
 
-# Offense count: 5
+# Offense count: 6
 # Configuration parameters: EnforcedStyle.
 # SupportedStyles: annotated, template, unannotated
 Style/FormatStringToken:
   Exclude:
     - 'app/helpers/order_cycles_helper.rb'
     - 'lib/open_food_network/sales_tax_report.rb'
+    - 'spec/features/admin/bulk_order_management_spec.rb'
+
+# Offense count: 928
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle.
+# SupportedStyles: always, never
+Style/FrozenStringLiteralComment:
```

Comment:
what is this new rule? 928 offenses.
maybe:
Style/FrozenStringLiteralComment:
  Enabled: false

Reply:
There are two reasons to enable frozen string literals:

1. Frozen strings can improve the performance by 20%.
  https://www.mikeperham.com/2018/02/28/ruby-optimization-with-one-magic-comment/
2. Frozen strings may become the default in future Ruby versions. This was planned for Ruby 3 but then abandoned because of compatibility concerns.
  https://bugs.ruby-lang.org/issues/11473#note-53

We can't add it blindly to every file. Some files depend on mutable strings. But we should work through this todo list as we modify files.

Reply:
so we should add the comment to as many files as possible?

Reply:
Yes. Well, it doesn't matter in files that don't use string literals. But every file that uses something like `hash["bla"]` would benefit from it.

Reply:
ok :+1:
I think the rule requires us to add the comment to every file in the codebase.